tests/test_hub.py

```diff
@@ -37,15 +36,6 @@ def _check_buckets_exist():
 
     assert eventually(_check_buckets_exist)
 
-    # check that buckets are listed correctly
```

Comment:
Please add this back. Removing this just breaks the test.

Reply:
the name property of the bucket in the storage service will no longer be structured according to a fixed schema. The comparison should therefore no longer take place.